packages/ui/src/containers/TermsOfUse/TermsOfUseConfirmModal/index.tsx

```diff
@@ -47,7 +47,9 @@ const TermsOfUseConfirmModal = ({ isOpen = false, onConfirm, onClose }: Props) =
         setTermsAgreement(true);
         onConfirm();
       }}
-      onClose={onClose}
+      onClose={() => {
+        onClose();
+      }}
```

Comment:
Maybe we can also revert this change

Reply:
Unfortunately not, as we keep the onClose as well as onConfirm method as the original MouseEventHandler. 
But the TermsOfUseConfirmModal onClose props type was (message?: string) => void;  Have to do the trick.